src/taco-cli/cli/utils/platformPluginCommandBase.ts

```diff
@@ -146,6 +153,9 @@ export class PlatformPluginCommandBase extends commands.TacoCommandBase {
             return cordovaWrapper.invokePlatformPluginCommand(self.name, self.cordovaCommandParams, data);
         })
             .then(function (): Q.Promise<any> {
+            return self.postInvokeStep(data);
```

Comment:
not sure if I follow, why do we need to execute add/remove related code like specsToPersist for "list" command? can we use existing pattern like "run" and invoke list via CordovaWrapper?